src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRunner.cs

```diff
@@ -259,7 +259,7 @@ public static async Task<RestoreSummary> CommitAsync(RestoreResultPair restoreRe
             var log = summaryRequest.Request.Log;
 
             // Commit the result
-            log.LogInformation(Strings.Log_Committing);
```

Comment:
This affects command-line restores as well right?  `dotnet restore` defaults to minimal verbosity and `msbuild /t:Restore` defaults to normal verbosity. Not sure if it really matters since its not taking something that was minimal to verbose.

Reply:
Yes, it affects commandline as well. 

The value of this one is definitely debatable, which is why is I moved it to Verbose. 

You'll already get a message about every single file that updated (assets file, or any of the nuget generated msbuild files).